R/as_gt.R

```diff
@@ -306,10 +301,10 @@ as_gt.gs_design <- function(
     display_bound = c("Efficacy", "Futility"),
     display_columns = NULL,
     display_inf_bound = FALSE,
-    full_alpha = 0.025,
```

Comment:
Is there any potential situation where a user would want to be able to manually override this?